ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/LedgerDao.scala

```diff
@@ -158,15 +156,12 @@ trait LedgerReadDao extends ReportsHealth {
 
 trait LedgerWriteDao extends ReportsHealth {
 
-  type LedgerOffset = Long
-
-  type ExternalOffset = LedgerString
+  type LedgerOffset
```

Comment:
I'm not sure there's much to be gained from making this abstract.

Reply:
I removed it. See dd158581e. Making it concrete clashes in `LedgerDao` when extending both traits.